io/file.go

```diff
@@ -164,12 +165,29 @@ func GetEmptyFiles(directory string) (filePaths []string, err error) {
 
 // FindFiles searches for files in the specified directory with the given fileName.
 // If fileName is empty, it retrieves all files in the directory.
+// If maxDepth is greater than 0, it limits the search to the specified depth.
 // It returns a slice of filePaths and an error if any.
-func FindFiles(directory, fileName string) (filePaths []string, err error) {
+func FindFiles(directory, fileName string, maxDepth int) (filePaths []string, err error) {
```

Comment:
Can we use [the 3rd-party implementation](https://github.com/Azure/draft/blob/5008e06cb5ebe6a3c54b30c40a09e34df3329ccd/pkg/reporeader/reporeader.go#L54)?

Reply:
> Can we use [the 3rd-party implementation](https://github.com/Azure/draft/blob/5008e06cb5ebe6a3c54b30c40a09e34df3329ccd/pkg/reporeader/reporeader.go#L54)?

This is used for unit-testing Azure/draft, I prefer not to use it. What is the benefit?

Reply:
> What is the benefit?

This is a very general function, and we don't need to implement it again and maintain it.
I'm fine with the current codes.